lib/Config.js

```diff
@@ -377,9 +377,15 @@ class Config {
                 'config: invalid host for localCache. host must be a string');
             assert(typeof config.localCache.port === 'number',
                 'config: invalid port for localCache. port must be a number');
+            if (config.localCache.password !== undefined) {
```

Comment:
@ThibaultRiviere Redis password is optional, so the config check will apply only when it's defined.